packages/api/test/e2e/api/promise-queries.spec.ts

```diff
@@ -313,12 +329,28 @@ describeE2E()('Promise e2e queries', (wsUrl): void => {
     });
 
     it('queries correct size', async (): Promise<void> => {
-      const size = await api.query.balances.freeBalance.size(keyring.alice.address);
+      const size = await api.query.balances.freeBalance.size(keyring.alice_stash.address);
 
       expect(size.toNumber()).not.toEqual(0);
     });
   });
+});
+
+// The following tests only run on nodes that support doubleMapType introduced by metadata v5
+describeE2E({
+  only: [
+    'docker-polkadot-master',
```

Comment:
we can add local here

Reply:
Yes, absolutely right, I kind of assumed that it includes `local`  by default even though I touched the code that defines it